arch/x86/crc_folding.c

```diff
@@ -39,8 +38,7 @@
         _mm_storeu_si128((__m128i *)s->crc0 + 1, xmm_crc1);\
         _mm_storeu_si128((__m128i *)s->crc0 + 2, xmm_crc2);\
         _mm_storeu_si128((__m128i *)s->crc0 + 3, xmm_crc3);\
-        _mm_storeu_si128((__m128i *)s->crc0 + 4, xmm_crc_part);\
-    } while (0);
+        _mm_storeu_si128((__m128i *)s->crc0 + 4, xmm_crc_part);
 
 ZLIB_INTERNAL void crc_fold_init(deflate_state *const s) {
     CRC_LOAD(s)
```

Comment:
CRC_LOAD is effectively dead code here now and could be removed. Variables below would need __m128i prefix though.